.circleci/config.yml

```diff
@@ -123,29 +123,34 @@ jobs:
     steps:
       - use_workspace
       - run:
-          name: Qlik docker login
-          command: echo $DOCKER_TOKEN | docker login -u "$DOCKER_USER" --password-stdin
+          name: Dockerhub login
+          command: |
+            echo $DOCKERHUB_PASSWORD | docker login -u "$DOCKERHUB_USERNAME" --password-stdin
+      - run:
+          name: Log into Github Docker Registry
+          command: |
+            echo $QLIK_DOCKER_DEV_PASSWORD | docker login -u $QLIK_DOCKER_DEV_USERNAME --password-stdin $QLIK_DOCKER_DEV_REGISTRY
```

Comment:
there are currently not credentials for a github user in circleci. (in dist-flow or qlik-oss)
and logging in to docker hub do not give access to ghcr.io/qlik-trial/api-compliance

Reply:
Ah okay, I will revert these changes then. Thanks @T-Wizard 

I will give it a try and add in the workflow `context: api-compliance` , I think that could give us access to the credentials without exposing them

Reply:
context: api-compliance give access to APICULTURIST_S3, APICULTURIST_GITHUB and APICULTURIST_TOKEN that is also needed.
You could try to login to ghcr.io/qlik-trial using APICULTURIST_GITHUB and APICULTURIST_TOKEN it could maybe work